src/main/java/uk/gov/hmcts/reform/cpo/service/mapper/CasePaymentOrderMapper.java

```diff
@@ -15,12 +15,14 @@
 
     @Mapping(target = "id", ignore = true)
     @Mapping(target = "createdTimestamp", ignore = true)
+    @Mapping(target = "historyExists", expression = "java(false)")
     CasePaymentOrderEntity toEntity(CreateCasePaymentOrderRequest createCasePaymentOrderRequest,
```

Comment:
we could have the create method in the Service class setting this property to false. Would make this less "hidden"

Reply:
if you follow @smathangi suggestion of using `boolean` in https://github.com/hmcts/cpo-case-payment-orders-api/pull/66#discussion_r619144832 then we can get rid of this? Since the default value is `false` for a boolean?

Reply:
Not relating this PR. But i think `@Mapping(target = "id", ignore = true)` is not needed. Can you try remove that please @KleoG